react/components/QuotesTableContainer.tsx

```diff
@@ -279,19 +287,17 @@ const QuotesTableContainer: FunctionComponent = () => {
       (permission: string) => permission.indexOf('access-quotes') >= 0
     )
   ) {
+    const message = !isAuthenticated ? (
```

Comment:
Can we switch the condition so it is less confusing?

Reply:
Done 👍